### PR TITLE
Supercharge your Kubernetes cluster with Kubeapps

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by caldav** [comment]
Demo started here: http://tutorials.ubuntu.com-pr-537.run.demo.haus/tutorial/kubeapps-on-canonical-kubernetes

File: tutorials/containers/kubeapps-on-canonical-kubernetes/kubeapps-on-canonical-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+status: published
+tags: kubeapps, kubeless, bitnami, kubernetes
+difficulty: 3
+published: 2017-12-07
+author: Sameer Naik <sameer@bitnami.com>
+feedback_url: https://github.com/kubeapps/kubeapps/issues
+
+---
+
+# Supercharge your Kubernetes cluster with Kubeapps
```
Comment (caldav) [inline tutorials/containers/kubeapps-on-canonical-kubernetes/kubeapps-on-canonical-kubernetes.md]: I'm not found of using "Supercharge" in the title, I think it's ok in the intro, but the title should be more down-to-earth and practical. Suggestions: "How to use Kubeapps on your Kubernetes cluster", "Install Kubeapps on your Kubernetes cluster"...

**Comment by sameersbn** [comment]
@caldav thanks the review. The title has been updated to **Install Kubeapps on your Kubernetes cluster**.

**Comment by caldav** [comment]
:+1:

**Pull request merged**